internal/version/version.go

```diff
@@ -2,7 +2,7 @@ package version
 
 var (
 	name    = "vogelkop"
-	version = "0.2.5"
+	version = "devel"
```

Comment:
Ideally we'd be updating this with each 'release'

Reply:
It wasn't documented and I missed it and I assumed goreleaser was taking care of it. goreleaser built binaries will have the right values and `go build` ones won't which should be a big red flag so is also pretty nice I think.